### ISSUE TITLE
wimpsim-reader - example.py error (Trac #2154)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2146]
In [source:IceCube/projects/wimpsim-reader/trunk/resources/examples/example.py#L19]

at line 19 there is a likely typing error:

```text
19	outfile = os.path.expandars("$I3_BUILD/wimpsim-reader/resources/example.i3.bz2")
```

I get this error:

```text
AttributeError: 'module' object has no attribute 'expandars'
```

I suppose it should be ```expandvars``` instead of ```expandars``` .

Migrated from https://code.icecube.wisc.edu/ticket/2154
```json
{
    "status": "closed", 
    "changetime": "2018-05-17T15:58:32", 
    "description": "In [source:IceCube/projects/wimpsim-reader/trunk/resources/examples/example.py#L19]\n\nat line 19 there is a likely typing error:\n\n\n{{{\n19\toutfile = os.path.expandars(\"$I3_BUILD/wimpsim-reader/resources/example.i3.bz2\")\n}}}\n\nI get this error:\n\n{{{\nAttributeError: 'module' object has no attribute 'expandars'\n}}}\n\n\nI suppose it should be {{{expandvars}}} instead of {{{expandars}}} .\n\n\n", 
    "reporter": "grenzi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1526572712892430", 
    "component": "combo simulation", 
    "summary": "wimpsim-reader - example.py error", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2018-05-17T15:18:51", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180517T15:55:49: 
* nega changed owner from "" to "nega"
* nega changed description from:

> In IceCube/projects/wimpsim-reader/trunk/resources/examples/example.py
> 
> at line 19 there is a likely typing error:
> 
> 
> {{{
> 19	outfile = os.path.expandars("$I3_BUILD/wimpsim-reader/resources/example.i3.bz2")
> }}}
> 
> I get this error:
> 
> {{{
> AttributeError: 'module' object has no attribute 'expandars'
> }}}
> 
> 
> I suppose it should be {{{expandvars}}} instead of {{{expandars}}} .
> 
> 
> ----
> 
> 
> In [http://software.icecube.wisc.edu/documentation/inspect/wimpsim_reader.html?highlight=i3wimpsim#I3WimpSimReader] we can read:
> 
> Param EndMJD:	Default = nan, MJD to end simulation; if unspecified: read everything
> 
> But if I try not to set it (and take the NAN default) I receive this error
> 
> 
> {{{
> ERROR (dataclasses): Calling with NAN not possible; will do nothing (I3Time.cxx:142 in void I3Time::SetModJulianTimeDouble(double))
> }}}
> 
> The same is for StartMJD.
> 
> I am not able to suggest a correction in this case
> 
> 
> 
> 
> 

to:

> In [source:IceCube/projects/wimpsim-reader/trunk/resources/examples/example.py#L19]
> 
> at line 19 there is a likely typing error:
> 
> 
> {{{
> 19	outfile = os.path.expandars("$I3_BUILD/wimpsim-reader/resources/example.i3.bz2")
> }}}
> 
> I get this error:
> 
> {{{
> AttributeError: 'module' object has no attribute 'expandars'
> }}}
> 
> 
> I suppose it should be {{{expandvars}}} instead of {{{expandars}}} .
> 
> 
> 


* nega commented:

> Note that not all examples are meant to be complete functional scripts. [source:IceCube/projects/wimpsim-reader/trunk/resources/examples/example.py example.py] is like that, as it's very incomplete (options aren't collected and parsed, NaN is used wrong, etc)


* nega changed status from "new" to "accepted"
* nega changed summary from "Two issues from wimpsim-reader example.py" to "wimpsim-reader - example.py error"

**Comment by icecube-trac** [comment]
Trac update at 20180517T15:58:32: 
* nega commented:

> In changeset 162732:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="162732"
> fix typo
> 
> this should be fleshed out into a full example
> 
> closes #2146
> ```


* nega changed resolution from "" to "fixed"
* nega changed status from "accepted" to "closed"
